content/en/load-testing/Manual/800-master-controller-cheatsheet.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Mastercontroller Cheat Sheet"
+linkTitle: "Cheat Sheet Mastercontroller"
+
+weight: 800
+type: docs
+
+description: >
+  You will find the usage instructions and a list of available commands below.
+---
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands) by executing all commands automatically.
+
+### Example
+
+- For displaying all the available command options you can run
+```powershell
+cd <XLT>\bin 
+>>./mastercontroller.cmd --help 
+```
+- This would make a load test run in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands), executing commands automatically in combination with [embedded mode](../310-test-execution/#embedded-mode).
+```powershell 
+cd <XLT>\bin
+>>./mastercontroller.cmd -auto -embedded -comment "Test Run" 
+```
+
+## List of Command Options
+- `-auto` is to be used for running a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands).
```

Comment:
A short summary of which steps the auto mode include and where it stops would be nice... and a hint to the -report option might be helpful

Reply:
Default Steps: upload, start, download - please update this info in https://xltdoc.xceptance.com/load-testing/manual/310-test-execution/#auto-mode